### PR TITLE
CHAIN-343: prompt user to add certain tokens to their wallet

### PR DESCRIPTION
**Pull request by bholzman**

### CODE REVIEW

File: web-ui/src/utils/addSymbolsToWallet.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  config.newSymbols?.map(async (sym) => {
+    if (
+      await window.ethereum?.request({
+        method: 'wallet_watchAsset',
+        params: {
+          type: 'ERC20',
+          options: {
+            address: sym.contractAddress,
+            // TODO: CHAIN-345 - improve cross-chain symbology
+            symbol: sym.name.replace(new RegExp('[0-9]+$', ''), ''),
```
Comment (bflood366) [inline web-ui/src/utils/addSymbolsToWallet.ts:14]: does a given ERC20 need to have the same address on all chains or does this just add for whatever the current chain id id. seems like the account configuration will return symbols for all chains if I understand correctly

**Review by bflood366 — APPROVED** [review]
looks great

File: web-ui/src/utils/addSymbolsToWallet.ts (hunk 1 above)
Comment (bholzman) [reply web-ui/src/utils/addSymbolsToWallet.ts:14]: ah, actually there's an optional chainId you can add here, let me add that

Comment (bholzman) [reply web-ui/src/utils/addSymbolsToWallet.ts:14]: going to fix this in CHAIN-345 since the chain id is easily accessible there

**Pull request merged**